### ISSUE TITLE
Wrap up test coverage + dev feedback for HFI Calculator Daily Table

### ISSUE THREAD

**Issue by conbrad** [bcgov/wps#1057]
**Describe the task**
Finish some tech QA work for HFI Calculator Daily Table. 

**Acceptance Criteria**
- [x] Dev pipeline should be passing
- [x] Code review completed
- [ ] Once the above is done, merged into main 

**Additional context**
- PR is currently open here: https://github.com/bcgov/wps/pull/1041
- Devs to decide if fire stations endpoint logic should be refactored(if yes make a new ticket for the refactoring)

**Comment by conbrad** [comment]
Unable to merge currently -- openshift builds are not working
